docs/glueops-platform-administrator/configuration/export-logs.md

```diff
@@ -0,0 +1,292 @@
+  name: 'glueops-json' # Must be unique among all objects of same type and match the name defined in ClusterFilter
+  namespace: glueops-core-fluent-operator  
+spec:
+  json:
+    timeFormat: "%Y-%m-%dT%H:%M:%S" # ISO 8601 standard
+    timeKey: "time"
+---
+```
+
+## Defining a custom lua script:
+
+```yaml title="Define custom lua script"
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    logging.glueops.dev: 'glueops-core-primary' # needs to match the label defined in: ClusterFluentBitConfig
+  name: 'glueops-core-containerd-lua-script-primary' # Must be unique among all objects of same type and needs to match the name in the ClusterFilter objects
+  namespace: glueops-core-fluent-operator 
+data:
+  containerd.lua: |
+    function containerd( tag, timestamp, record)
+            if(record["logtag"]~=nil)
+            then
+            timeStr = os.date("!*t",  timestamp["sec"])
+            t = string.format("%4d-%02d-%02dT%02d:%02d:%02d.%sZ",
+            timeStr["year"], timeStr["month"], timeStr["day"],
+            timeStr["hour"], timeStr["min"], timeStr["sec"],
+            timestamp["nsec"]);
+            record["time"] = t;
+            record["log"] = record["message"];
+            record["message"] =  nil;
+            return 1, timestamp, record
+            else
+            return 0,timestamp,record
+            end
+    end
+---
+```
+
+
+## Filtering/Parsing:
+
+```yaml title="Define filters and parsers"
+apiVersion: fluentbit.fluent.io/v1alpha2
+kind: ClusterFilter
+metadata:
+  labels:
+    logging.glueops.dev: 'glueops-core-primary' # needs to match the label defined in: ClusterFluentBitConfig
+  name: glueops-core-filter-primary # Must be unique among all objects of same type
+  namespace: glueops-core-fluent-operator
+spec:
+  filters:
+    - lua:
+        call: containerd
+        script:
+          key: containerd.lua
+          name: glueops-core-containerd-lua-script-primary # needs to match the name of the script defined earlier
+        timeAsTable: true
+    - parser:
+        alias: parsedata
+        keyName: log
+        parser: 'glueops-json' # see name used in ClusterParser definition
+        reserveData: true
+    - kubernetes:
+        annotations: true
+        kubeCAFile: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+        kubeTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+        kubeURL: 'https://kubernetes.default.svc:443'
+        labels: true
+        mergeLogTrim: true
+    - nest:
+        addPrefix: kubernetes_
+        nestedUnder: kubernetes
+        operation: lift
+    - modify:
+        rules:
+          - remove: stream
+          - remove: kubernetes_pod_id
+    - nest:
+        nestUnder: kubernetes
+        operation: nest
+        removePrefix: kubernetes_
+        wildcard:
+          - kubernetes_*
+  match: kube.*
+---
+```
+
+## Defining Outputs:
```

Comment:
are you planning to add other output options, or would it be better to call this section "Defining Datadog Output"

Reply:
Planned to eventually add other outputs

Reply:
Updated the name to be datadog specifically